Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM quay.io/azavea/scala:2.10.6
+FROM openjdk:7-jre
```

Comment:
Ah yeah, that makes sense re: Alpine

I wasn't able to find anything official, but it appears others have used it. I don't think it would be a problem because the bytecode generated with older JDKs can be run on newer JVMs I don't think upgrading would hurt anything here.

If someone is using our latest scala container to build a spark application they'd be compiling it with Java 8 and it wouldn't be able to run here with Java 7 though.

FWIW - there are [instructions](https://spark.apache.org/docs/1.6.2/building-spark.html#running-java-8-test-suites) for running java 8 tests on the 1.6.2 version of spark, but I'm not sure what that means for their released version. I would guess that the packaged version of spark was compiled with 7.